contracts/PoolManager.sol

```diff
@@ -391,6 +392,13 @@ contract PoolManager is IPoolManager, Fees, NoDelegateCall, ERC1155, IERC1155Rec
         emit HookFeeUpdated(id, newHookFees);
     }
 
+    function setDynamicFee(PoolKey memory key) external {
```

Comment:
I know you're following pattern of protocol/hook fees, but `set*` feels like the wrong verb here as generally that implies the params include the new value. I'd lean towards `update*`